x-pack/plugins/observability_solution/infra/public/components/asset_details/utils/get_data_stream_types.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EntityDataStreamType } from '@kbn/observability-shared-plugin/common';
+
+export function isMetricsSignal(dataStreamTypes: EntityDataStreamType[] = []) {
```

Comment:
Do you think naming it `hasMetricsSignal` would make more sense?

Reply:
I updated it from `hasMetrics` to `isMetrics` based on [this comment](https://github.com/elastic/kibana/pull/195378#discussion_r1795346714) in the PR.

And now I see I missed the "Signal" part 😅 so I'll update it to `isMetricsSignal`.